lib/developer_portal/lib/liquid/drops/request.rb

```diff
@@ -20,6 +20,8 @@ def request_uri
         @request.original_url
       end
 
+      alias original_url request_uri
```

Comment:
why was that?

Reply:
Mmm, this was originally related to https://github.com/3scale/porta/blob/cba325cfc80bfb78743ac9d010a0e530e4007559/lib/developer_portal/lib/liquid/url_helper_hacks.rb#L9-L13

First I had to replace `.request_uri` to `.original_url` because `.request_uri` was missing or something, but this caused the need to create this alias.
Now I am not quite sure this is needed, but probably it is.

Reply:
I don't see `.original_url` being called anywhere, maybe it is not needed after all?

Reply:
I'd say its [defined](https://github.com/rails/rails/blob/90ab51b910080a54d4804cf704e1f714f2be1ef1/actionpack/lib/action_dispatch/http/request.rb#L273-L275) and [used](https://github.com/rails/rails/blob/90ab51b910080a54d4804cf704e1f714f2be1ef1/actionpack/lib/action_dispatch/http/request.rb#L446) from `ActionDispatch::Request`. So probably still needed.

Reply:
Thanks for the references @jlledom

Reply:
Although not clear how `Drops::Base` is related to a rails request :) but fine